### DISCUSSION TITLE
网络稳定性差，github都推不上去，不满足工作要求，要求退款

### DISCUSSION THREAD

**Discussion by wjt123** [getlantern/lantern-forum-cn#1086] [General]

**Comment by wjt123** [comment]
![Screenshot_20241217_114738_com eg android AlipayGphone](https://github.com/user-attachments/assets/41d642d6-e0d4-4296-83df-6bfa90c79700)

**Comment by MaxMKMax** [comment]
@wjt123 您好, 请您也提供我账号邮箱一下

**Comment by MaxMKMax** [comment]
@wjt123 在帮您提交退款之前需要您先帮个忙我们才能进行退款. 我们也想找出原因为什么网络稳定性差.

您可以帮助我们在应用程序中提交报告您的问题吗？您可以访问Lantern应用程序并提交log/ 报告。

go to Support (转到支持)
select Report an Issue (选择报告问题)
choose issue type (选择问题类型)
include additional details (包括额外的详细信息)
hit Send Report (点击发送报告)

当您在app提交问题报告后您会收到邮件. 邮件标题上有 8 位数号码. 请回来这里留言您的log/ticket号码. 因为我需要拿您的反馈报告号码来和科技团队沟通您遇到的问题. 不然我无法提交您遇到的问题给科技团队. 不好意思, 需要您的帮忙. 谢谢!

有了这些资料，我们科技团队便可以尽快解决问题。您可以在留言区给我知道。谢谢。
